enterprise/server/image_converter/image_converter.go

```diff
@@ -189,7 +188,12 @@ func (c *imageConverter) convertImage(ctx context.Context, req *rgpb.ConvertImag
 		layerDesc := layerDesc
 
 		eg.Go(func() error {
-			newLayer, err := c.imageConverterClient.ConvertLayer(egCtx, &rgpb.ConvertLayerRequest{
+			convConn, err := grpc_client.DialTarget(*imageConverterBackend)
```

Comment:
do we need a `convConn.Close()` somewhere?

Reply:
Good catch!